srml/elections-phragmen/src/lib.rs

```diff
@@ -373,6 +380,10 @@ decl_module! {
 
 		/// What to do at the end of each block. Checks if an election needs to happen or not.
 		fn on_initialize(n: T::BlockNumber) {
+			if !DidMigrate::exists() {
```

Comment:
```suggestion
			if !DidMigrate::get() {
```

`false` is the default value.